compose.yml

```
@@ -0,0 +1,304 @@
+      - ./.compose/ssl/certs:/certs:rw
+
+  # ---------------------------------------------------------------------------
+  # Reverse proxy
+  # Dynamic config is injected by compose.override.yml (dev) or
+  # compose.deploy.yml (deploy) via volume override.
+  # ---------------------------------------------------------------------------
+  traefik:
+    image: traefik:v2.11
+    restart: unless-stopped
+    depends_on:
+      mkcert: { condition: service_completed_successfully }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)